include/RAJA/index/IndexSetUtils.hpp

```diff
@@ -106,17 +102,15 @@ RAJA_INLINE void getIndicesConditional(CONTAINER_T& con,
  *
  ******************************************************************************
  */
-template <typename CONTAINER_T, typename SEGMENT_T, typename CONDITIONAL>
+template<typename CONTAINER_T, typename SEGMENT_T, typename CONDITIONAL>
 RAJA_INLINE void getIndicesConditional(CONTAINER_T& con,
                                        const SEGMENT_T& seg,
                                        CONDITIONAL conditional)
 {
   CONTAINER_T tcon;
-  forall<seq_exec>(seg,
-    [&](typename CONTAINER_T::value_type idx) {
-      if (conditional(idx)) tcon.push_back(idx);
-    }
-  );
+  forall<seq_exec>(seg, [&](typename CONTAINER_T::value_type idx) {
+    if (conditional(idx)) tcon.push_back(idx);
```

Comment:
We're allowing no-curly braces in single line conditionals?

Reply:
I think we may have to move to clang-format 15 according to this: https://github.com/llvm/llvm-project/commit/77e60bc42c48e16d646488d43210b1630cd4db49